community/working-groups.md

```diff
@@ -0,0 +1,57 @@
+
+Note: "ET" refers to Eastern Time.
+
+| Working Group                    | Slack channel       | Meetings                                 | Contact                                                         |
+| -------------------------------- | ------------------- | ---------------------------------------- | --------------------------------------------------------------- |
+| Base Julia CI                    | `#ci-dev`           | Every other Tuesday at 2:15pm ET         | Elliot Saba (@staticfloat), Dilum Aluthge (@DilumAluthge)       |
+| [GPU][gpu]                       | `#gpu`              | Every other Monday at 11am ET            | Tim Besard (@maleadt), Valentin Churavy (@vchuravy)             |
+| High-performance computing (HPC) | `#hpc`              | Fourth Tuesday of each month at 2pm ET   | Valentin Churavy                                                |
+| Machine learning                 | `#machine-learning` | Every other Tuesday at 12pm ET           | Kyle Daruwalla (@darsnack), Lorenz Ohly (@lorenzoh)             |
+| Multithreading                   | `#multithreading`   | First Wednesday of each month at 10am ET | Jameson Nash (@vtjnash)                                         |
+| [Pkg][pkg-jl]                    | `#pkg-dev`          | Occasional Tuesdays at 2:15pm ET         | Kristoffer Carlsson (@KristofferC), Fredrik Ekre (@fredrikekre) |
+| Release management               | `#releases`         | [none]                                   | Kristoffer Carlsson, Alex Arslan (@ararslan)                    |
+
+Note: the machine learning working group also uses the `#`
+
+[gpu]: https://github.com/JuliaGPU
+[pkg]: https://github.com/JuliaLang/Pkg.jl
+
+### Triage call
+
+In addition to working groups, the Julia community also holds regular triage calls
+to discuss open issues and pull requests. Unlike working groups, the triage calls
+tend to discuss a wider range of topics. Triage calls are held online and are open
+to anyone. Please check Slack for the meeting link.
+
+| Name    | Slack channel | Meetings                          | Contact       |
+| ------- | ------------- | --------------------------------- | ------------- |
+| Triage  | `#triage`     | Every other Thursday at 2:15pm ET | Jeff Bezanson |
+
+### Community meetings
+
+
+| Name                             | Slack channel       | Meetings                                 | Contact                                                         |
+| -------------------------------- | ------------------- | ---------------------------------------- | --------------------------------------------------------------- |
+| JuliaCon committee               |                     |                                          |                                                                 |
+| Julia gender inclusive           |                     |                                          |                                                                 |
```

Comment:
@ztangent Should say whether she wants this listed here and if so what to fill in.

Reply:
Discussing this with the other organizers, also tagging @kimauth and @rowanxshi just so they're in the loop.

Reply:
I'll remove it for now, and I can add it back once a decision is made.